From: =?utf-8?q?Jonas_=C3=85dahl?= <jadahl@gmail.com>
Date: Wed, 6 Oct 2021 10:00:43 +0200
Subject: main: Leak the GJS context and ShellGlobal

There are many crash-on-exit happening as a side effect of destroying
the GJS context. Work around these until we have a better solution by
leaking them.

Origin: vendor, Fedora
Forwarded: not-needed, workaround for issues fixed more intrusively in 44.beta
Bug-Debian: https://bugs.debian.org/1038972
---
 src/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/main.c b/src/main.c
index 29275cd..2311a74 100644
--- a/src/main.c
+++ b/src/main.c
@@ -589,9 +589,11 @@ main (int argc, char **argv)
 
   shell_profiler_shutdown ();
 
+#if 0
   g_debug ("Doing final cleanup");
   _shell_global_destroy_gjs_context (shell_global_get ());
   g_object_unref (shell_global_get ());
+#endif
 
   return ecode;
 }
